### PR TITLE
feat(media-app): add docs for custom media apps

### PR DESCRIPTION
**Pull request by nickangtc**
- add docs for custom media apps

Don't deploy until ready for the world to know about custom media app integration.

### CODE REVIEW

**Comment by nickangtc** [comment]
Added modifications that reflect latest demo movie app

File: docs/integrations/custom-media-app.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+```
+
+> **TIP**: Your base template filename will vary depending on how your developer named it.
+
+> Common names are index and theme, with file extensions like .html, .erb, .ejs, .liquid (eg. index.ejs or theme.liquid)
+
+Once jQuery is available, paste the following script to the same file below jQuery. This will load the Metisa script that is needed to send and receive data to/from Metisa.
+
+```html
+<script src="https://metisa-sdk.s3-ap-southeast-1.amazonaws.com/js/v0.0/browser.js"></script>
```
Comment (jyek) [inline docs/integrations/custom-media-app.md]: this should be metisa.js and needs to be updated on S3?

this also needs to be updated on metisa-js/docs

File: docs/integrations/custom-media-app.md [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+
+## <a id="prerequisites"></a> Prerequisites
+
+To integrate your media app with Metisa:
+
+1. **You must have a Metisa account.** Custom integrations are available to all Metisa users, whether you're on a Free or Paid tier. Sign up [here](https://askmetisa.com/create).
+2. **You must have basic technical know-how.** You will be fine if you're comfortable navigating files and inserting code blocks, and some light coding with guidance.
+3. **Your app must use a template rendering engine.** EJS, Liquid, Django are some examples. This installation rides on your app's template tags to obtain data to send to Metisa servers.
+
+On the last point, if you are a developer, you may be able to customize the way data is sent to Metisa  servers by fiddling with the [Metisa JavaScript SDK](https://github.com/altitudelabs/metisa-js).
```
Comment (jyek) [inline docs/integrations/custom-media-app.md]: 1. extra space --> Metisa  servers
2. metisa-js docs also need a bit of cleanup
"Metisa empowers businesses ..." is incomplete on `https://github.com/altitudelabs/metisa-js/blob/master/doc/README.md`

File: docs/integrations/custom-media-app.md (hunk 1 above)
Comment (nickangtc) [reply docs/integrations/custom-media-app.md]: 👍 Ok will update here, shouldn't have let it slip!

I've already updated metisa-js/docs for `metisa.js`

**Pull request merged**